components/prize_pool/commons/prize_pool_placement.lua

```diff
@@ -382,7 +266,7 @@ function Placement:_getLpdbData(...)
 		lpdbData.objectName = self.parent:_lpdbObjectName(lpdbData, ...)
 		if Opponent.isTbd(opponent.opponentData) then
 			_tbd_index = _tbd_index + 1
-			lpdbData.objectName = lpdbData.objectName .. '_' .. _tbd_index
```

Comment:
Is this a mistake? (https://github.com/Liquipedia/Lua-Modules/pull/2485#discussion_r1091970583)

Reply:
yes